examples/java_service/src/clj/java_service_example/java_service.clj

```diff
@@ -8,20 +8,20 @@
   (meaning-of-life-fn [this]))
 
 (defservice java-service
-            JavaService
-            []
+  JavaService
+  []
             ;; Service functions are implemented in a java `ServiceImpl` class
```

Comment:
There's now too much indentation on this line.  I wonder if there's a way to get cljfmt to fix these automatically.